.travis.yml

```diff
@@ -7,4 +7,5 @@ rvm:
   - 2.2.0
   - 2.2.1
   - 2.2.2
+  - 2.3.0
```

Comment:
Any reason not to add 2.3.1, 2.3.3, and/or 2.4.0 while you're adding stuff?  I would leave out 2.3.2 because it's broken.